snuba/api.py

```diff
@@ -353,6 +354,21 @@ def sdk_distribution(validated_body, timer):
     )
 
 
+@application.route('/subscriptions', methods=['POST'])
+def create_subscription():
+    return json.dumps({'subscription_id': uuid4().hex}), 202, {'Content-Type': 'application/json'}
+
+
+@application.route('/subscriptions/<uuid>/renew', methods=['POST'])
+def renew_subscription(uuid):
+    return 'ok', 202, {'Content-Type': 'text/plain'}
+
+
+@application.route('/subscriptions/<uuid>', methods=['DELETE'])
+def delete_subscription(uuid):
+    return 'ok', 202, {'Content-Type': 'text/plain'}
```

Comment:
Should this be 204?

Reply:
IMO: no, since deletions also happen asynchronously (all of these API calls just send messages to Kafka, the consumer handles the deletion.)